src/Symfony/Bridge/Propel1/phpunit.xml.dist

```diff
@@ -7,9 +7,7 @@
          bootstrap="vendor/autoload.php"
 >
     <php>
-        <!-- Disable E_USER_DEPRECATED until 3.0 -->
-        <!-- php -r 'echo -1 & ~E_USER_DEPRECATED;' -->
-        <ini name="error_reporting" value="-16385"/>
+        <ini name="error_reporting" value="-1" />
```

Comment:
True, I just tested it. It uses whatever is configured in your own php.ini which by default probably does not enable all errors because PHPs default is not so strict. So I agree it should be set explicitly to -1.